share/OpenMS/CHEMISTRY/Custom_RNA_modifications.tsv

```diff
@@ -34,6 +34,11 @@ ambiguous dimethylguanosine ("m2,2G" or "m_Gm")	mmG?		G	Ó	&Oacute;	C12O5N5H17	3
 ambiguous trimethylguanosine ("m2,2,7G" or "m2,7Gm")	mmmG?		G	Ô	&Ocirc;	C13O5N5H21	327.1543	327.3404	m2,2,7G m2,7Gm
 N4-acetylcytidine or 5-formyl-2'-O-methylcytidine	ac4C/f5Cm?		C	M	M	C11O6N3H15	285.0961	285.257	ac4C f5Cm
 3-(3-amino-3-carboxypropyl)uridine or 5-carboxymethylaminomethyl-2'-O-methyluridine	acp3U/cmnm5Um?		U	X	X	C13O8N3H19	345.1172	345.3096	acp3U cmnm5Um
+deoxyadenosine	dA	dA	A			C10O3N5H13	251.1018	251.2424
+deoxycytidine	dC	dC	C			C9O4N3H13	227.0906	227.2176
+deoxyguanosine	dG	dG	G			C10O4N5H13	267.0968	267.2418
+deoxyuridine	dU	dU	U			C9O5N2H12	228.0746	228.2024
+thymidine	dT	dT	U			C10O5N2H14	242.0903	242.2290
```

Comment:
U?

Reply:
Yup. We treat the DNA bases as modified (de-oxygenated) RNA ribobases. Since thymidine doesn't have a true analog in the four RNA ribobases we treat it as methyl-deoxy-uridine, hence U here.

Reply:
makes sense... just wanted to confirm that it is ot a copy paste thign